firmware/STN1110.c

```diff
@@ -29,56 +29,102 @@
 #include <stdlib.h>
 #include "settings.h"
 #include "system.h"
+#include "modp_numtoa.h"
```

Comment:
I assume you are doing this to avoid using libc's printf and friends in any way.